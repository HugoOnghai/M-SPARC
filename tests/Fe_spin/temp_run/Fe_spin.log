
                            < M A T L A B (R) >
                  Copyright 1984-2019 The MathWorks, Inc.
                  R2019a (9.6.0.1072779) 64-bit (glnxa64)
                               March 8, 2019

 
To get started, type doc.
For product information, visit www.mathworks.com.
 

 Reading .inpt file ...


<INPT>
# $ cat Fe_spin.inpt
# nprocs: 24

SPIN_TYP: 1
#LATVEC:
#0.5 0.5 0.0
#0.0 0.5 0.5
#0.5 0.0 0.5
CELL: 5.42 5.42 5.42
MESH_SPACING: 0.15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 2 2 2
EXCHANGE_CORRELATION: GGA_PBE
ELEC_TEMP_TYPE: fermi-dirac
ELEC_TEMP: 3157.73
TOL_SCF: 1e-6
NSTATES: 35
CALC_STRESS: 1
PRINT_FORCES: 1
PRINT_ATOMS: 0
<\INPT>

 Reading .ion file ...
 Number of atom types : 1
 Total number of atoms: 2
 Default atomic mass for Fe is 55.845000
 pseudo_dir # 1 = FeGGA.psp8

 COORD:
                   0                   0                   0
   2.710000000000000   2.710000000000000   2.710000000000000

  17.326022764070000

 kpoint grid before symmetry:
   0.289814820441863   0.289814820441863   0.289814820441863
  -0.289814820441863   0.289814820441863   0.289814820441863
   0.289814820441863  -0.289814820441863   0.289814820441863
  -0.289814820441863  -0.289814820441863   0.289814820441863
   0.289814820441863   0.289814820441863  -0.289814820441863
  -0.289814820441863   0.289814820441863  -0.289814820441863
   0.289814820441863  -0.289814820441863  -0.289814820441863
  -0.289814820441863  -0.289814820441863  -0.289814820441863

 kpoint grid after symmetry:
   0.289814820441863   0.289814820441863  -0.289814820441863
  -0.289814820441863   0.289814820441863  -0.289814820441863
   0.289814820441863  -0.289814820441863  -0.289814820441863
  -0.289814820441863  -0.289814820441863  -0.289814820441863

## Chebyshev polynomial degree not provided, finding npl ...
## Based on the mesh size, npl is set to: 42
## Poisson tolerance not provided, choosing poisson_tol ...
## poisson_tol is set to: 1.000e-08
## Pseudocharge tolerance not provided, choosing pseudocharge_tol ...
## pseudocharge_tol is set to: 1.000e-08
 Creating differentiation matrices ...
 Done. (1.355 sec)

 Estimated memory usage:
 Total:  927.62 MB
 orbitals            :  797.34 MB
 sparse matrices     :  120.31 MB
 global-size vectors :    4.98 MB
 mixing histories    :    4.98 MB

 Finding rb for Fe ...
 rb = {2.409 2.409 2.409}, int_b = -16.000301265725401, err_rb = 3.013e-04
 rb = {2.559 2.559 2.559}, int_b = -16.000259037641023, err_rb = 2.590e-04
 rb = {2.710 2.710 2.710}, int_b = -16.000229628525581, err_rb = 2.296e-04
 rb = {2.861 2.861 2.861}, int_b = -16.000207307802818, err_rb = 2.073e-04
 rb = {3.011 3.011 3.011}, int_b = -16.000190148248748, err_rb = 1.901e-04
 rb = {3.162 3.162 3.162}, int_b = -16.000176242005203, err_rb = 1.762e-04
 rb = {3.312 3.312 3.312}, int_b = -16.000164057733102, err_rb = 1.641e-04
 rb = {3.463 3.463 3.463}, int_b = -16.000154250059897, err_rb = 1.543e-04
 rb = {3.613 3.613 3.613}, int_b = -16.000145133502951, err_rb = 1.451e-04
 rb = {3.764 3.764 3.764}, int_b = -16.000137818978740, err_rb = 1.378e-04
 rb = {3.914 3.914 3.914}, int_b = -16.000127684527822, err_rb = 1.277e-04
 rb = {4.065 4.065 4.065}, int_b = -16.000127684527822, err_rb = 1.277e-04
 rb = {4.216 4.216 4.216}, int_b = -16.000126208586092, err_rb = 1.262e-04
 rb = {4.366 4.366 4.366}, int_b = -16.000118157113874, err_rb = 1.182e-04
 rb = {4.517 4.517 4.517}, int_b = -16.000108758478916, err_rb = 1.088e-04
 rb = {4.667 4.667 4.667}, int_b = -16.000090532131960, err_rb = 9.053e-05
 rb = {4.818 4.818 4.818}, int_b = -16.000083004424024, err_rb = 8.300e-05
 rb = {4.968 4.968 4.968}, int_b = -16.000078647426939, err_rb = 7.865e-05
 rb = {5.119 5.119 5.119}, int_b = -16.000077704333233, err_rb = 7.770e-05
 rb = {5.269 5.269 5.269}, int_b = -16.000070500521062, err_rb = 7.050e-05
 rb = {5.420 5.420 5.420}, int_b = -16.000058814232958, err_rb = 5.881e-05
 rb = {5.571 5.571 5.571}, int_b = -16.000062367378305, err_rb = 6.237e-05
 rb = {5.721 5.721 5.721}, int_b = -16.000060815642431, err_rb = 6.082e-05
 rb = {5.872 5.872 5.872}, int_b = -16.000050585892367, err_rb = 5.059e-05
 rb = {6.022 6.022 6.022}, int_b = -15.999993443447035, err_rb = 6.557e-06
 rb = {6.173 6.173 6.173}, int_b = -16.000001364069306, err_rb = 1.364e-06
 rb = {6.323 6.323 6.323}, int_b = -15.999999730046618, err_rb = 2.700e-07
 rb = {6.474 6.474 6.474}, int_b = -16.000000038387689, err_rb = 3.839e-08
 rb = {6.624 6.624 6.624}, int_b = -15.999999997233713, err_rb = 2.766e-09
 rb = {6.624 6.624 6.624}

 ###############################################################
 Relaxation step number: 1 

 Starting pseudocharge generation and self energy calculation...
 Integration b = 31.999999994467

 Integration b_ref = 31.999999999969

======================================
 Net initial magnetization is:  5.999999998962657 
======================================

 ****************************************
 *          Eself_ref = 914.360880       *
 ****************************************
 Done. (1.105113 s)
 Time for b calculation: 1.120 seconds.

 Starting calculating nonlocal projectors ...
 Done. (0.378773 s)

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Starting SCF iteration...
 AAR converged to a relative residual of 9.7583e-09 in 72 iterations.
 Poisson problem took 0.888447s
 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 1 
 ============================================= 
 Fermi energy = 4.343176
 This SCF iteration took 152.219 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 2 
 ============================================= 
 Fermi energy = 0.349874
 This SCF iteration took 141.028 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 3 
 ============================================= 
 Fermi energy = 0.232759
 This SCF iteration took 143.967 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 4 
 ============================================= 
 Fermi energy = 0.231371
 AAR converged to a relative residual of 5.1275e-09 in 61 iterations.
 Poisson problem took 0.519591s
 Escc = 10.83695983
 Etot = -392.39381272
 Eatom = -196.19690636
======================================
Net magnetization in this iteration is:  3.214340526955256 
======================================
 Error in SCF iteration: 1.9631e-01 
 This SCF iteration took 137.208 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  2 
 ========================= 
 Fermi energy = 0.288021
 AAR converged to a relative residual of 2.42471e-09 in 67 iterations.
 Poisson problem took 0.543690s
 Escc = 1.30789040
 Etot = -392.70413966
 Eatom = -196.35206983
======================================
Net magnetization in this iteration is:  1.062150200154269 
======================================
 Error in SCF iteration: 7.1467e-02 
 This SCF iteration took 137.621 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  3 
 ========================= 
 Fermi energy = 0.269702
 AAR converged to a relative residual of 1.80339e-09 in 61 iterations.
 Poisson problem took 0.518131s
 Escc = 0.49862973
 Etot = -392.74407654
 Eatom = -196.37203827
======================================
Net magnetization in this iteration is:  0.447249870187314 
======================================
 Error in SCF iteration: 4.3270e-02 
 This SCF iteration took 138.169 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  4 
 ========================= 
 Fermi energy = 0.255202
 AAR converged to a relative residual of 4.25733e-09 in 61 iterations.
 Poisson problem took 0.482717s
 Escc = 0.24944474
 Etot = -392.76263908
 Eatom = -196.38131954
======================================
Net magnetization in this iteration is:  0.057337273901542 
======================================
 Error in SCF iteration: 2.5045e-02 
 This SCF iteration took 137.226 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  5 
 ========================= 
 Fermi energy = 0.257913
 AAR converged to a relative residual of 3.96273e-09 in 55 iterations.
 Poisson problem took 0.465612s
 Escc = 0.12919561
 Etot = -392.76536506
 Eatom = -196.38268253
======================================
Net magnetization in this iteration is:  0.052779620759821 
======================================
 Error in SCF iteration: 1.2177e-02 
 This SCF iteration took 138.459 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  6 
 ========================= 
 Fermi energy = 0.261400
 AAR converged to a relative residual of 5.90474e-09 in 55 iterations.
 Poisson problem took 0.466423s
 Escc = -0.17353969
 Etot = -392.76637107
 Eatom = -196.38318554
======================================
Net magnetization in this iteration is:  0.024183212050644 
======================================
 Error in SCF iteration: 4.2393e-03 
 This SCF iteration took 136.932 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  7 
 ========================= 
 Fermi energy = 0.259473
 AAR converged to a relative residual of 7.03748e-09 in 49 iterations.
 Poisson problem took 0.387141s
 Escc = 0.02765003
 Etot = -392.76661557
 Eatom = -196.38330779
======================================
Net magnetization in this iteration is:  0.001956738997471 
======================================
 Error in SCF iteration: 1.4405e-03 
 This SCF iteration took 137.467 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  8 
 ========================= 
 Fermi energy = 0.259568
 AAR converged to a relative residual of 9.81173e-09 in 46 iterations.
 Poisson problem took 0.389120s
 Escc = 0.04208345
 Etot = -392.76662412
 Eatom = -196.38331206
======================================
Net magnetization in this iteration is: -0.002921405785688 
======================================
 Error in SCF iteration: 9.1905e-04 
 This SCF iteration took 136.277 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  9 
 ========================= 
 Fermi energy = 0.259955
 AAR converged to a relative residual of 3.24891e-09 in 43 iterations.
 Poisson problem took 0.357767s
 Escc = -0.00304127
 Etot = -392.76662989
 Eatom = -196.38331495
======================================
Net magnetization in this iteration is: -0.000112923443155 
======================================
 Error in SCF iteration: 1.1525e-04 
 This SCF iteration took 137.589 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 10 
 ========================= 
 Fermi energy = 0.259860
 AAR converged to a relative residual of 9.95824e-09 in 35 iterations.
 Poisson problem took 0.296694s
 Escc = 0.00136874
 Etot = -392.76662993
 Eatom = -196.38331496
======================================
Net magnetization in this iteration is:  0.000018571121599 
======================================
 Error in SCF iteration: 2.7436e-05 
 This SCF iteration took 137.271 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 11 
 ========================= 
 Fermi energy = 0.259873
 AAR converged to a relative residual of 5.9054e-09 in 31 iterations.
 Poisson problem took 0.256493s
 Escc = 0.00002389
 Etot = -392.76662992
 Eatom = -196.38331496
======================================
Net magnetization in this iteration is:  0.000027167924081 
======================================
 Error in SCF iteration: 7.3410e-06 
 This SCF iteration took 136.984 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 12 
 ========================= 
 Fermi energy = 0.259873
 AAR converged to a relative residual of 9.88168e-09 in 28 iterations.
 Poisson problem took 0.237517s
 Escc = -0.00014128
 Etot = -392.76662992
 Eatom = -196.38331496
======================================
Net magnetization in this iteration is:  0.000001536056285 
======================================
 Error in SCF iteration: 2.8440e-06 
 This SCF iteration took 138.081 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 13 
 ========================= 
 Fermi energy = 0.259871
 AAR converged to a relative residual of 4.35678e-09 in 25 iterations.
 Poisson problem took 0.207258s
 Escc = 0.00001161
 Etot = -392.76662992
 Eatom = -196.38331496
======================================
Net magnetization in this iteration is:  0.000001343973469 
======================================
 Error in SCF iteration: 8.7802e-07 
 This SCF iteration took 136.724 s.


 Finished SCF iteration in 13 steps!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 

 **********************************************************
 *          Energy per unit cell = -392.766629924 Ha.       *
 *          Energy per atom = -196.383314962 Ha.            *
 **********************************************************

 Starting atomic force calculation ... 
 local force calculation: 1.546 s
 ***********************************************************
 *                      Atomic Force                       *
 ***********************************************************
 Drift free forces (Ha/Bohr):
   1.0e-05 *

   0.096892112682769   0.287736319005243   0.399733496646923
  -0.096892112682769  -0.287736319005243  -0.399733496646923

 Max magnitude of forces (Ha/Bohr):     5.019632846298554e-06


 Time for calculating forces: 5.858759 s.

["Stress in GPa"


]  42.501091032584270  -0.000063901171538   0.000038403914321
  -0.000063901171538  42.501133373613762   0.000011505910965
   0.000038403914321   0.000011505910965  42.501413789186657


 Time for calculating stress: 16.818255 s.

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Final atomic positions (Cartesian) are as follows:
  0.000000 	  0.000000 	  0.000000 
  2.710000 	  2.710000 	  2.710000 

 Run-time of the program: 2251.504894 seconds

ans = 

  struct with fields:

                  cell_typ: 1
                   lat_vec: [3x3 double]
                  metric_T: [3x3 double]
                    grad_T: [3x3 double]
                    lapc_T: [3x3 double]
                      Jacb: 1
                        L1: 5.420000000000000
                        L2: 5.420000000000000
                        L3: 5.420000000000000
                        Nx: 36
                        Ny: 36
                        Nz: 36
                         N: 46656
                      ecut: -1
              mesh_spacing: 0.150000000000000
                   kptgrid: [4x3 double]
                  kptshift: [1x3 double]
                      nkpt: [2 2 2]
                     tnkpt: 4
                      wkpt: [4x1 double]
                        BC: 2
                       BCx: 0
                       BCy: 0
                       BCz: 0
                      isBS: 0
                   lattice: 'undefined'
                   SCF_tol: 1.000000000000000e-06
                       Nev: 35
               poisson_tol: 1.000000000000000e-08
          pseudocharge_tol: 1.000000000000000e-08
                       Cst: 27.211386245987999
              Ha_Bohr3_GPa: 2.942101569650548e+04
                        kB: 3.166811563455600e-06
               elec_T_type: 0
                      Temp: 3.157730000000000e+03
                       bet: 1.000006412214051e+02
                       npl: 42
                       FDn: 6
                    rc_ref: 0.500000000000000
              max_relax_it: 100
                dbg_switch: 0
                        xc: 2
                 Nelectron: 32
                     n_typ: 1
                TimeRevSym: 1
                  spin_typ: 1
                     nspin: 2
              CheFSI_Optmz: 0
          chefsibound_flag: 0
               FixRandSeed: 0
                rhoTrigger: 4
                 NetCharge: 0
                 MAXIT_SCF: 100
                 MINIT_SCF: 3
             MAXIT_POISSON: 1000
            accuracy_level: -1
     target_force_accuracy: -1
    target_energy_accuracy: -1
                 TOL_RELAX: 5.000000000000000e-04
               TOL_LANCZOS: 0.010000000000000
               precond_tol: 2.266697530864198e-05
               precond_kTF: 1
            precond_thresh: 0.250000000000000
            precond_fitpow: 2
          precond_resta_q0: 1.360000000000000
          precond_resta_Rs: 2.760000000000000
            MixingVariable: 1
             MixingPrecond: 0
                  Pf_guess: []
             MixingHistory: 7
           MixingParameter: 0.300000000000000
     MixingParameterSimple: 0.300000000000000
            PulayFrequency: 1
          PulayRestartFreq: 0
                    TWtime: 1.000000000000000e+09
                 RelaxFlag: 0
                 RelaxMeth: 'LBFGS'
            max_dilatation: 1.200000000000000
            TOL_RELAX_CELL: 0.010000000000000
                    MDFlag: 0
               RestartFlag: 0
                    MDMeth: 'NVE'
                     MD_dt: 1
                  MD_Nstep: 0
                     ion_T: -1
                thermos_TF: -1
              ion_elec_eqT: 1
              ion_vel_dstr: 2
                NLCG_sigma: 0.500000000000000
                     qmass: 1
                 L_history: 20
               L_finit_stp: 0.005000000000000
                  L_maxmov: 0.200000000000000
               L_autoscale: 1
                 L_lineopt: 1
                   L_icurv: 1
                   FIRE_dt: 1
                 FIRE_mass: 1
               FIRE_maxmov: 0.200000000000000
               Calc_stress: 1
                 Calc_pres: 0
            PrintForceFlag: 1
          PrintAtomPosFlag: 0
          PrintAtomVelFlag: 1
         PrintElecDensFlag: 0
            PrintEigenFlag: 0
                PrintMDout: 1
             PrintRelaxout: 1
              Printrestart: 1
           Printrestart_fq: 1
                       xin: 0
                       yin: 0
                       zin: 0
                     twist: 0
                  filename: 'Fe_spin'
            inputfile_path: '.'
                        XC: 'GGA_PBE'
                       Atm: [1x1 struct]
                     n_atm: 2
                     Atoms: [2x3 double]
           mvAtmConstraint: [2x3 double]
                    IsFrac: 0
                    IsSpin: 1
            is_default_psd: 0
                  temp_tol: 1.000000000000000e-12
                 xc_rhotol: 1.000000000000000e-14
                Atoms_init: [2x3 double]
                  lat_uvec: [3x3 double]
                        dx: 0.150555555555556
                        dy: 0.150555555555556
                        dz: 0.150555555555556
                        dV: 0.003412639060357
                        w2: [1x7 double]
                        w1: [1x7 double]
                         W: [46656x1 double]
                    occfac: 1
                Relax_iter: 1
              elecgs_Count: 1
               StressCount: 1
                    amu2au: 1.822888485332371e+03
                    fs2atu: 41.341373336493000
                 isOutl_11: [468x1 logical]
                 isOutr_11: [468x1 logical]
                      I_11: [468x1 double]
                     II_11: [468x1 double]
                      V_11: [468x1 double]
                 isOutl_22: [468x1 logical]
                 isOutr_22: [468x1 logical]
                      I_22: [468x1 double]
                     II_22: [468x1 double]
                      V_22: [468x1 double]
                 isOutl_33: [468x1 logical]
                 isOutr_33: [468x1 logical]
                      I_33: [468x1 double]
                     II_33: [468x1 double]
                      V_33: [468x1 double]
                 G_JOutl_1: [559872x1 logical]
                 G_JOutr_1: [559872x1 logical]
                     G_J_1: [559872x1 double]
                     G_I_1: [559872x1 double]
                     G_V_1: [559872x1 double]
                 G_JOutl_2: [559872x1 logical]
                 G_JOutr_2: [559872x1 logical]
                     G_J_2: [559872x1 double]
                     G_I_2: [559872x1 double]
                     G_V_2: [559872x1 double]
                 G_JOutl_3: [559872x1 logical]
                 G_JOutr_3: [559872x1 logical]
                     G_J_3: [559872x1 double]
                     G_I_3: [559872x1 double]
                     G_V_3: [559872x1 double]
                   Lap_std: [46656x46656 double]
                    grad_1: [46656x46656 double]
                    grad_2: [46656x46656 double]
                    grad_3: [46656x46656 double]
                LapPreconL: [46656x46656 double]
                LapPreconU: [46656x46656 double]
              memory_usage: 972684296
               staticfname: 'Fe_spin.static'
                 suffixNum: 0
                  outfname: 'Fe_spin.out'
                  parallel: 0
                         b: [46656x1 double]
                     b_ref: [46656x1 double]
                    rho_at: [46656x3 double]
                     Eself: 2.424348031075413e+02
                 Eself_ref: 9.143608796775824e+02
                       V_c: [46656x1 double]
                 PosCharge: 31.999999994467380
                 NegCharge: -31.999999994467380
                    E_corr: -0.002672487483551
                       rho: [46656x3 double]
                      Atom: [1x2 struct]
                       phi: [46656x1 double]
                      e_xc: [46656x1 double]
                 dvxcdgrho: [46656x3 double]
                       Vxc: [46656x2 double]
                      Veff: [46656x2 double]
                         X: [93312x7 double]
                         F: [93312x7 double]
          mixing_hist_fkm1: [93312x1 double]
          mixing_hist_xkm1: [93312x1 double]
                       psi: [46656x35x8 double]
    upper_bound_guess_vecs: [46656x8 double]
                    EigVal: [35x8 double]
                  lambda_f: 0.259871312947185
                       occ: [35x8 double]
                    Etotal: -3.927666299235857e+02
                     Eband: -1.813968839900548e+02
                       Exc: -25.405605009403583
                    Exc_dc: -33.122244264941820
                  Eelec_dc: 23.355237059838405
                      Eent: -0.004158264888252
                      Escc: 1.161100558379376e-05
                      netM: 1.343973469403181e-06
                   S_Debug: [1x1 struct]
                     force: [2x3 double]
                 abs_force: [2x1 double]
                    Stress: [3x3 double]
                  Pressure: -42.501212731794894
             stress_dimred: [3x3 double]

