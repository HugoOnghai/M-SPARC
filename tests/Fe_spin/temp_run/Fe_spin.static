Atomic forces (Ha/Bohr):
      0.0000009689       0.0000028774       0.0000039973
     -0.0000009689      -0.0000028774      -0.0000039973
Stress (GPa):
  4.2501091033E+01  -6.3901171538E-05   3.8403914321E-05 
 -6.3901171538E-05   4.2501133374E+01   1.1505910965E-05 
  3.8403914321E-05   1.1505910965E-05   4.2501413789E+01 
