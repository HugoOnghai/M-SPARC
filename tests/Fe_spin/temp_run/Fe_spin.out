***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 11-May-2021  Start time: 21:08:39                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 5.420000 5.420000 5.420000 
FD_GRID: 36 36 36
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 2 2 2
KPOINT_SHIFT: 5.000000e-01 5.000000e-01 5.000000e-01
SPIN_TYP: 1
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.009999936
CHEB_DEGREE: 42
NSTATES: 35
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: Fe_spin.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.150556 (Bohr)
Number of symmetry adapted k-points:  4
Output printed to                  :  Fe_spin.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Fe 16
Pseudopotential                    :  FeGGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.62 6.62 6.62
Number of atoms of type 1          :  2
Estimated total memory usage       :  927.62 MB
========================================================================================
                            Self Consistent Field (SCF#1)                     
========================================================================================
Iteration     Free Energy (Ha/atom)    Magnetization     SCF Error        Timing (sec)
1            -1.9619690636E+02         3.2143E+00        1.963E-01        574.419
2            -1.9635206983E+02         1.0622E+00        7.147E-02        137.619
3            -1.9637203827E+02         4.4725E-01        4.327E-02        138.166
4            -1.9638131954E+02         5.7337E-02        2.505E-02        137.224
5            -1.9638268253E+02         5.2780E-02        1.218E-02        138.455
6            -1.9638318554E+02         2.4183E-02        4.239E-03        136.930
7            -1.9638330779E+02         1.9567E-03        1.441E-03        137.464
8            -1.9638331206E+02        -2.9214E-03        9.191E-04        136.275
9            -1.9638331495E+02        -1.1292E-04        1.153E-04        137.586
10           -1.9638331496E+02         1.8571E-05        2.744E-05        137.269
11           -1.9638331496E+02         2.7168E-05        7.341E-06        136.981
12           -1.9638331496E+02         1.5361E-06        2.844E-06        138.070
13           -1.9638331496E+02         1.3440E-06        8.780E-07        136.720
Total number of SCF: 13    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -1.9638331496E+02 (Ha/atom)
Total free energy                  : -3.9276662992E+02 (Ha)
Band structure energy              : -1.8139688399E+02 (Ha)
Exchange correlation energy        : -2.5405605009E+01 (Ha)
Self and correction energy         : -2.4243747560E+02 (Ha)
Entropy*kb*T                       : -4.1582648883E-03 (Ha)
Fermi level                        :  2.5987131295E-01 (Ha)
Net Magnetization                  :  1.3439734694E-06 
Average force                      :  5.0196328463E-06 (Ha/Bohr)
Maximum force                      :  5.0196328463E-06 (Ha/Bohr)
Time for force calculation         :  5.859 (sec)
Pressure                           : -4.2501212732E+01 (GPa)
Max stress                         :  4.2501413789E+01 (GPa)
Time for stress calculation        :  16.820 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  2251.505 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
